packages/jellyfish-api-core/__tests__/category/spv/listAnchors.test.ts

```diff
@@ -1,64 +1,47 @@
-import { MasterNodeRegTestContainer, ContainerGroup, GenesisKeys } from '@defichain/testcontainers'
-import { ContainerAdapterClient } from '../../container_adapter_client'
+import { TestingGroup } from '@defichain/jellyfish-testing'
+import { GenesisKeys, MasterNodeRegTestContainer } from '@defichain/testcontainers'
 
 describe('Spv', () => {
-  const group = new ContainerGroup([
-    new MasterNodeRegTestContainer(GenesisKeys[0]),
-    new MasterNodeRegTestContainer(GenesisKeys[1]),
-    new MasterNodeRegTestContainer(GenesisKeys[2])
-  ])
-
-  const clients = [
-    new ContainerAdapterClient(group.get(0)),
-    new ContainerAdapterClient(group.get(1)),
-    new ContainerAdapterClient(group.get(2))
-  ]
+  const tGroup = TestingGroup.create(3, i => new MasterNodeRegTestContainer(GenesisKeys[i]))
```

Comment:
once https://github.com/DeFiCh/jellyfish/pull/621/files#r704077129 is done, you can call `TestingGroup.create(3)` here.